ansible/roles/sidekiq/handlers/main.yml

```diff
@@ -0,0 +1,12 @@
+---
+  - name: monit stop all
+    command: monit stop all
+    sudo: true
+
+  - name: monit reload
+    command: monit reload
+    sudo: true
+
+  - name: monit start all
```

Comment:
yeah shouldn't these handlers be defined in the monit config? and is there a way to chain them?